modules/accredited_representative_portal/app/controllers/accredited_representative_portal/v0/form21a_controller.rb

```diff
@@ -22,14 +31,33 @@ def submit
 
       attr_reader :parsed_request_body
 
+      def schema
+        # NOTE: This doesn't reject any extra attributes not found in the schema. If
+        # we want that the schema needs to have { "additionalProperties" => false }
+        VetsJsonSchema::SCHEMAS[FORM_ID.upcase]
```

Comment:
I see the 21a schema isn't requiring properties such that `'{}'` is valid when it is submitted to this endpoint. That behavior is incorrect and it should be updated in the schema, but maybe we can just add a comment for now since we're moving on from this work stream.

Reply:
Really good catch on that.